codegen/arrai/grpc_client.arrai

```diff
@@ -0,0 +1,98 @@
+# https://github.com/anz-bank/sysl/blob/master/pkg/sysl/sysl.proto
+# Sysl data structure to handle
+\sysl (
+    grpcclient: \a \name \proj_location
```

Comment:
Some of the variable names could be a little more descriptive.

Reply:
Ack